patch/action.go

```diff
@@ -8,16 +8,17 @@ import (
 	"github.com/urfave/cli"
 )
 
-func patch(patchPath, kstPath *string) func(c *cli.Context) error {
+// TODO to be implemented
+// patch - Patch all yaml files in the given path with the new Docker image reference
+func patch(c *cli.Context) error {
```

Comment:
In my opinion (I have changed my mind) we shouldn't implement the patch command in korp. Instead we should document in the the README how the user can use the various options and subcommands of `kustomize` to get what they need.

Reply:
I agree. We can create a specific PR to completely remove the patch feature.

Reply:
Sorry I was following the flow of emails. I approved and already merged the feature branch removing the patch feature.